tests/backup_helper.go

```diff
@@ -203,6 +207,84 @@ const (
 	Parallel
 )
 
+// Type for customResourceObjects created by backup
+type customResourceObjectDetails struct {
+	Group         string
+	Version       string
+	Resource      string
+	SkipResources []string // If this list is populated that particular CR will not be considered for cleanup
+}
+
+// List of all the CRs to be considered for cleanup
+var crListMap = map[string]customResourceObjectDetails{
+	"applicationbackups": customResourceObjectDetails{
+		Group:         "stork.libopenstorage.org",
+		Version:       "v1alpha1",
+		Resource:      "applicationbackups",
+		SkipResources: []string{},
+	},
+	"applicationbackupschedules": customResourceObjectDetails{
```

Comment:
does this CR get deleted?

Reply:
Yes, there are few schedulepolicies CRs which are not deleted, we have added exception for the same